ouroboros-consensus-cardano/src/ouroboros-consensus-cardano/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -528,7 +528,7 @@ instance CardanoHardForkConstraints c
       , (NodeToClientV_15, CardanoNodeToClientVersion11)
       ]
 
-  latestReleasedNodeVersion _prx = (Just NodeToNodeV_10, Just NodeToClientV_14)
+  latestReleasedNodeVersion _prx = (Just NodeToNodeV_11, Just NodeToClientV_15)
```

Comment:
Do we need to update this? If Conway is not imminent we should skip this change.

Reply:
That's probably a question for @coot. I'm not familiar with the release schedule.

Reply:
@dnadales I am ok with including it, but first we need to publish new version of `ouroboros-network` which includes input-output-hk/ouroboros-network#4256.  I merged it today, I'll release new version soon.

Reply:
I am confused why this PR didn't require a new version of `ouroboros-network-api`.

Reply:
Peer Sharing is on `V_11` so this would mean not needing to enable `TestEnableDevelopmentNetworkProtocols` flag to get it, right? (Of course one would still need to manually enable the `PeerSharing` flag)

Reply:
@bolt12 thanks for reminding me that, this means we need to use `V_12` for handshake changes, it will be safer as I am not sure if things will get into `8.0`.

Reply:
@jprider63 in input-output-hk/ouroboros-network#4538 I added `NodeToNodeV_12` and `NodeToClientV_16`, you'll need to change it here.

Reply:
You will also need to add:

*  `(NodeToNodeV_11, CardanoNodeToNodeVersion7)` to `supportedNodeToNodeVersions`
*  `(NodeToClientV_16, CardanoNodeToClientVersion11)` to `supportedNodeToClientVersions`

Reply:
If we use `ouroboros-network-api-0.5.0.0` (which is published to CHaP) then we don't need to change anything here.

Reply:
`ouroboros-network-api-0.5.0.0`, `ouroboros-network-0.5.0.2` were released in this PR:
https://github.com/input-output-hk/cardano-haskell-packages/pull/279